test/unit/helpers/node_mock.js

```diff
@@ -50,6 +50,33 @@ module.exports = {
         },
         node_info: { network: `test-net` }
       }
+    },
+    async net_info() {
```

Comment:
going forward we probably shouldn't work with the `node_mock.js` but use the new more unity approach to testing and add mocked values to the tests individually.